script/mafia

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -eu
 
+MAFIA_HOME=${MAFIA_HOME:-$HOME/.mafia}
+
```

Comment:
@markhibberd I think I like what you have better as it only mentions `MAFIA_HOME` once

Can you explain what it means? why have the `:` at the start?